pages/terraform/quickstart.mdx

```diff
@@ -13,21 +13,21 @@ categories:
   - devtools
 ---
 
-[Terraform](https://www.terraform.io/) is an Infrastructure as Code (IaC) software environment tool that allows you to easily build and maintain your cloud resources. You can use it to safely and efficiently deploy Scaleway resources in a declarative way.
+[Terraform](https://www.terraform.io/) or [OpenTofu](https://opentofu.org/) is an Infrastructure as Code (IaC) software environment tool that allows you to easily build and maintain your cloud resources. You can use it to safely and efficiently deploy Scaleway resources in a declarative way.
 
-Check out the [Scaleway Terraform registry](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs) documentation for more information.
+Check out the [Scaleway Terraform/OpenTofu registry](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs) documentation for more information.
```

Comment:
In this part please mention the OpenTofu registry with a different site https://search.opentofu.org/provider/opentofu/scaleway/latest

Reply:
```suggestion
Check out the [Scaleway Terraform](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs) or [OpenTofu](https://search.opentofu.org/provider/opentofu/scaleway/latest) registry documentation for more information.
```